one.ablaze.floorp.yml

```diff
@@ -0,0 +1,45 @@
+app-id: one.ablaze.floorp
+name: Floorp
+runtime: org.freedesktop.Platform
+runtime-version: 21.08
+sdk: org.freedesktop.Sdk
+
+command: floorp
+
+finish-args:
+  - --share=ipc
+  - --socket=pcsc  
+  - --socket=x11
+  - --share=network
+  - --socket=pulseaudio
+  - --filesystem=~/.mozilla
```

Comment:
should this be `--persist` instead?

Reply:
fixed in https://github.com/flathub/flathub/pull/3103/commits/91776956fc4b0520a58ae9a7e854960060d49b29